api-generation/src/main/scala/ai/h2o/sparkling/api/generation/scala/MOJOModelTemplate.scala

```diff
@@ -136,6 +158,37 @@ object MOJOModelTemplate
       .mkString("\n\n")
   }
 
+  def generateOutputParameterAssignments(outputs: Seq[Parameter]): String = {
+    outputs
+      .map { output =>
+        val h2oName = output.h2oName
+        val swName = output.swName
+        val value = output.dataType.getSimpleName match {
+          case "boolean" => s"""outputSection.get("$h2oName").getAsBoolean()"""
```

Comment:
can the `outputSection...` prefix be extracted here?

Reply:
outputSection is not a prefix in all cases (see [case "TwoDimTableV3"](https://github.com/h2oai/sparkling-water/pull/2692/files/9c82aed66ab0b3664a679e512906f727c88e44e6#diff-9b82cfbfe9fbcf2977a5d0bab5833436bd3709f353ab4c38f6988df6fa48df9eR176))